### PR TITLE
Fix a typo in syntax-extensions.rst

### PR DESCRIPTION
**Pull request by pi8027**
<!-- Pointers to relevant developer documentation:

Contributing guide: https://github.com/coq/coq/blob/master/CONTRIBUTING.md

Test-suite: https://github.com/coq/coq/blob/master/test-suite/README.md

Changelog: https://github.com/coq/coq/blob/master/doc/changelog/README.md

Building the doc: https://github.com/coq/coq/blob/master/doc/README.md
Sphinx: https://github.com/coq/coq/blob/master/doc/sphinx/README.rst
doc_gram: https://github.com/coq/coq/blob/master/doc/tools/docgram/README.md

Overlays: https://github.com/coq/coq/blob/master/dev/ci/user-overlays/README.md

### CODE REVIEW

**Comment by pi8027** [comment]
BTW, I have an impression that what this sentence means is slightly difficult to understand because the example below does not include any "symbol starting with a single quote".

**Review by jfehrle — APPROVED** [review]
Thanks for the PR!

**Comment by jfehrle** [comment]
Do you want to add an example?

**Comment by pi8027** [comment]
> Do you want to add an example?

No, I don't, because I'm not sure whether adding such an example really improves the readability or not.

Let me try to rephrase the issue: when I was reading this paragraph, I was assuming each pair of adjacent sentences had some sort of connection. Then, I failed to find the connection between "In the same vein, ..." and "Here is an example."

Anyway, I just wanted to share how I failed to understand things, so that someone may come up with a better explanation later.

**Comment by Alizter** [comment]
You could also just change it to "Here is an example of using single quotes:".

**Comment by jfehrle** [comment]
> Anyway, I just wanted to share how I failed to understand things, so that someone may come up with a better explanation later.

If we don't fix it now when you notice it, we probably will lose track of your suggestion.

How about this?  If you like it I'll add it to the PR.

```
The other elements of the string (such as ``/\``) are the *symbols*, which must appear
literally when the notation is used.

Identifiers enclosed in single quotes are treated as symbols and thus
lose their role as parameters. For example:

.. coqtop:: in

   Notation "'IF' c1 'then' c2 'else' c3" := (c1 /\ c2 \/ ~ c1 /\ c3) (at level 200, right associativity).

Symbols that start with a single quote with 3 or more characters quote must be quoted.
For example the symbol `'abc` is represented by `''abc'`` in the notation string. 
```

**Comment by pi8027** [comment]
@jfehrle That's much better. I agree with your suggestion. Thanks!

**Comment by jfehrle** [comment]
@Zimmi48 Would you review and assign?  Thanks.

**Comment by Zimmi48** [comment]
@jfehrle it looks like there was a bad rebase, reintroducing stuff that was removed in a separate PR that was recently merged. Please check.

**Comment by jfehrle** [comment]
@Zimmi48 Got it.

File: doc/sphinx/user-extensions/syntax-extensions.rst [hunk 1]
```diff
@@ -59,17 +59,19 @@ abbreviation has the form of an ordinary applicative expression;
...
-lose their role of parameters. In the same vein, every symbol of at
-least 3 characters and starting with a simple quote must be quoted
-(then it starts with two single quotes). Here is an example.
+lose their role as parameters. For example:
 
 .. coqtop:: in
 
    Notation "'IF' c1 'then' c2 'else' c3" := (c1 /\ c2 \/ ~ c1 /\ c3) (at level 200, right associativity).
 
+Symbols that start with a single quote with 3 or more characters must be quoted.
```
Comment (Alizter) [inline doc/sphinx/user-extensions/syntax-extensions.rst:72]: Maybe clearer?
```suggestion
Symbols that start with a single quote with 3 or more characters must be single quoted.
```

**Review by Alizter — APPROVED** [review]

**Review by Zimmi48 — APPROVED** [review]
I'm fine with the current version or with Ali's suggestion. Let me know when this is ready to merge.

**Comment by jfehrle** [comment]
Updated, please merge, thanks.

**Comment by Zimmi48** [comment]
@coqbot merge now

**Pull request merged**